src/App.js

```diff
@@ -1,30 +1,51 @@
 import React, { Component } from "react";
-import Dashboard from './Dashboard';
-import "./App.css"
-import ReconnectingWebSocket from 'reconnecting-websocket';
+import Dashboard from "./Dashboard";
+import "./App.css";
+import ReconnectingWebSocket from "reconnecting-websocket";
 import Alert from '@material-ui/lab/Alert';
 import Snackbar from '@material-ui/core/Snackbar';
+const axios = require("axios");
+
+
+const corsOptions = {
+  origin: "*",
+};
 
 export class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
       events: [],
+      destinations: [],
       open: false,
-    }
+    };
   }
 
+  fetchServices = () => {
+    axios
+      .post("http://sockeye.default.20.190.7.108.xip.io/queryservices",{},corsOptions)
```

Comment:
```suggestion
      .post("/queryservices",{},corsOptions)
```

might just drop the host here for a post.